.github/workflows/docker-integration-test.yaml

```diff
@@ -0,0 +1,38 @@
+name: docker-integration-test
+on: [push]
+jobs:
+  docker-integration-test:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
```

Comment:
The integration test builds the docker image and pushes it to this local registry.